cloudflare.php

```diff
@@ -13,7 +13,6 @@
     exit;
 }
 
-define('CLOUDFLARE_MIN_WP_VERSION', '3.4');
```

Comment:
https://plugins.svn.wordpress.org/akismet/trunk/akismet.php

This is a good example of first file.